wallet_support.json

```diff
@@ -116,6 +116,21 @@
       "uri": "https://github.com/sbddesign"
     }
   },
+  {
+    "wallet": {
+      "name": "BTCPay Server",
+      "uri": "https://btcpayserver.org/"
+    },
+    "scans_bip21": "no",
+    "recognizes_lightning": "?",
+    "creates_bip21": "no",
+    "description": "",
```

Comment:
```suggestion
    "description": "BTCPayServer does not scan QR codes. The primary use case for BTCPay would be to create BIP21 URIs instead of having an on-chain and lightning tab, which would require more industry support first.",
```